postgres/experiment.yaml

```diff
@@ -1,11 +1,11 @@
 apiVersion: redskyops.dev/v1alpha1
 kind: Experiment
 metadata:
-  name: postgres-example
+  name: postgres-example-test-cost-metric
```

Comment:
probably want to leave this at the example name

Reply:
Nice catch.